README.md

```diff
@@ -1,6 +1,6 @@
 <!-- Readme template from https://github.com/othneildrew/Best-README-Template/blob/master/BLANK_README.md-->
 
-<a name="readme-top"></a>
+<a name="readmeTop"></a>
```

Comment:
Turned out to not be a necessary name change. I thought the anchor tag wasn't working but it was just how the preview was displaying.